database/migrations/20220512200302_add-userid.sql

```diff
@@ -0,0 +1,68 @@
+CREATE TABLE old_users (
+    username TEXT PRIMARY KEY,
+    password TEXT NOT NULL,
+    prefs BLOB NOT NULL DEFAULT '{}',
+    claimed_invite TEXT NOT NULL UNIQUE,
+    roles TEXT[] NOT NULL DEFAULT 'User',
+    picture INTEGER UNIQUE,
+
+    FOREIGN KEY(claimed_invite) REFERENCES invites(id),
+    FOREIGN KEY(picture) REFERENCES assets(id)
+);
+
+INSERT INTO old_users (username, password, prefs, claimed_invite, roles, picture) SELECT * FROM users;
+
+
+CREATE TABLE old_progress (
+    id INTEGER NOT NULL,
+    user_id TEXT NOT NULL,
+    delta INTEGER NOT NULL,
+    media_id INTEGER NOT NULL,
+    populated INTEGER NOT NULL,
+
+    PRIMARY KEY (id),
+    FOREIGN KEY(media_id) REFERENCES _tblmedia (id) ON DELETE CASCADE ON UPDATE CASCADE,
+    FOREIGN KEY(user_id) REFERENCES old_users(username) ON DELETE CASCADE
+);
+
+INSERT INTO old_progress (id, user_id, delta, media_id, populated) SELECT * FROM progress;
+
+DROP TABLE users;
+DROP TABLE progress;
```

Comment:
Looking at the SQlite3 docs, sqlite supports ```ALTER TABLE RENAME TO``` queries. I think we can just use that to rename the table to `old_table`, then create a new table and do the insertion